packages/graph/hash_graph/lib/graph/src/store/postgres/resolve/context.rs

```diff
@@ -87,4 +97,22 @@ impl<C: AsClient> PostgresContext for PostgresStore<C> {
             .await
             .attach_printable("could not read links")?)
     }
+
+    async fn read_active_links_by_source(
+        &self,
+        entity_id: EntityId,
+    ) -> Result<links::RecordStream, QueryError> {
+        Ok(links::read_active_links_by_source(&self.client, entity_id)
+            .await
+            .attach_printable("could not read links")?)
```

Comment:
You think we should be clear about direction here?
```suggestion
            .attach_printable("could not read outgoing links")?)
```